scripts/test_config.yaml

```diff
@@ -1,54 +1,58 @@
 # config.yaml
+potentials:
+  - provider: "physics-ml"
+    model_name: "MACE"
+    precision: 64
+    position_scaling: 10.0
+    output_scaling: 4.184 * 627
+    model_path: "/guardowl/tests/data/physics-ml/mace"
```

Comment:
This path will fail as it is not treated as relative.

Also when running the mace test, the `model_artefacts/module_checkpoint.ckpt` file is missing from the data directory